src/components/Shifting/ShiftDetails.tsx

```diff
@@ -40,555 +34,247 @@ export default function ShiftDetails(props: { id: string }) {
-                    {t("covid_19_cat_gov")}:{" "}
-                  </span>{" "}
-                  {consultation.category || "-"}
-                </p>
-                <p>
-                  <span className="font-semibold leading-relaxed">
-                    {t("referred_to")}:{" "}
-                  </span>
-                  {data.assigned_facility_external ||
-                    data.assigned_facility_object?.name ||
-                    "--"}
-                </p>
-                <p>
-                  <span className="font-semibold leading-relaxed">
-                    {t("reason_for_referral")}:{" "}
-                  </span>{" "}
-                  {data.reason || "--"}
-                </p>
-                <p>
-                  <span className="font-semibold leading-relaxed">
-                    {t("treatment_summary")}:{" "}
-                  </span>{" "}
-                  {consultation.treatment_plan || "-"}
-                </p>
+        <div>
+          <div className="flex flex-col lg:flex-row justify-between mt-2 rounded-lg border p-4">
+            <div className="grid grid-cols-1 gap-4 lg:grid-cols-2 lg:gap-8 w-full lg:w-auto">
+              <div className="border-b-2 pb-2 lg:border-b-0 lg:border-r-2 lg:pr-4 lg:pb-0">
+                <div className="text-sm font-medium leading-5 text-secondary-700">
+                  {t("created")} on
+                </div>
+                <div className="mt-1 whitespace-pre-wrap text-sm leading-5 text-gray-700 font-semibold">
+                  <RecordMeta
+                    time={data?.created_date}
+                    user={data?.created_by_object}
+                    inlineUser={true}
+                    inlineClassName={"flex-wrap"}
+                  />
+                </div>
               </div>
-            </div>
-
-            <div className="my-6 border-b-2"></div>
 
-            <div className="text-center">
-              <p className="text-sm text-muted-foreground">
-                {t("approved_by_district_covid_control_room")}
-              </p>
-              <p className="text-sm">{t("auto_generated_for_care")}</p>
+              <div className="mt-2 lg:mt-0">
+                <div className="text-sm font-medium leading-5 text-secondary-700">
+                  {t("last_edited")}
+                </div>
+                <div className="mt-1 whitespace-pre-wrap text-sm leading-5 text-gray-700 font-semibold">
+                  <RecordMeta
+                    time={data?.modified_date}
+                    user={data?.last_edited_by_object}
+                    inlineUser={true}
+                    inlineClassName={"flex-wrap"}
+                  />
+                </div>
+              </div>
             </div>
-          </CardContent>
-        </Card>
-      </PrintPreview>
-    );
-  };
 
-  if (loading) {
-    return <Loading />;
-  }
+            <div className="flex flex-col gap-4 mt-4 lg:mt-0 w-full lg:flex-row lg:w-auto">
```

Comment:
<img width="840" alt="image" src="https://github.com/user-attachments/assets/a3e82f53-0178-46df-ba0f-1f3f131cc3da" />

- Adjust the font sizes of both update status/details and referral letters.
- Adjust alignment.